### DISCUSSION TITLE
How to make voice channel type youtube togethet ?

### DISCUSSION THREAD

**Discussion by OMAR-3OOV** [discord-jda/JDA#1662] [Questions and Help]
I just only one of **discord bot** using this future where I can find it?

**Comment by RedcodesDev** [comment]
This feature is currently in development [#1628](https://github.com/DV8FromTheWorld/JDA/pull/1628)

**Comment by Splayfery** [comment]
You can only launch an activity at a vc channel by creating an vc channel invite which has to be clicked by an user. After the user clicked on that link the activity automatically starts in the channel and other members can join. You can create this invite with:
`vc.createInvite().setTargetApplication("id")`
The id of YouTube Together is "880218394199220334"!
